### PR TITLE
In tests, use stronger password.

### PR DESCRIPTION
**Pull request by mauritsvanrees**
Otherwise the tests in test_env.py fail during setup when the January 2020 hotfix is merged to CMFPlone.
The password strength is tested in more places then.

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
See for example [here on Jenkins](https://jenkins.plone.org/job/pull-request-5.1/218/testReport/junit/plone.api.tests.test_env/TestPloneApiEnv/test_adopt_fewers_role/)

**Comment by mauritsvanrees** [comment]
- To me there is no sense in running all those Jenkins tests for this simple change
- Travis mostly succeeds after a minor update to `travis.yml`. 
- But the Travis test for the docs fails, which is totally unrelated to this PR.

So my inclination is to merge this. But I would like someone else to agree.

**Comment by mauritsvanrees** [comment]
Issue for the failing Travis docs: https://github.com/plone/plone.api/issues/437

File: 3021.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+In tests, use stronger password.  [maurits]
```
Comment (gforcada) [inline 3021.bugfix]: Is that a path typo, or done in purpose? :thinking: this file should go to the `news` folder, isn't it?

Comment (mauritsvanrees) [reply 3021.bugfix]: Oops. :-) Fixed.

**Comment by ale-rt** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**